libraries/AP_Networking/AP_Networking_CAN.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include "AP_Networking.h"
+#include <AP_HAL/utility/Socket.h>
+
+class AP_Networking_CAN {
+public:
+    void start(uint8_t bus_mask);
```

Comment:
```suggestion
    void start(const uint8_t bus_mask);
```
Could add `const` on the public part of the class.

Reply:
changed, although it really doesn't matter